.buildkite/pipeline.yml

```diff
@@ -912,6 +912,36 @@ steps:
           - "sphere_held_suarez_rhoe_hightop"
           - "gpu_held_suarez_rhoe_hightop"
 
+      - label: "GPU: GPU dry baroclinic wave"
+        key: "target_gpu_implicit_baroclinic_wave"
+        command:
+          - mkdir -p target_gpu_implicit_baroclinic_wave
+          - >
+            nsys profile --trace=nvtx,cuda --output=target_gpu_implicit_baroclinic_wave/report
+            julia --color=yes --project=examples examples/hybrid/driver.jl
+            --config_file ${GPU_CONFIG_PATH}target_gpu_implicit_baroclinic_wave.yml
+        artifact_paths: "target_gpu_implicit_baroclinic_wave/*"
+        agents:
+          slurm_gpus: 1
+          slurm_time: 23:00:00
```

Comment:
Does it take this long?

Reply:
No. This doesn't take that long. I just removed the `slurm_time` entry.